src/keygen/participant.rs

```diff
@@ -76,7 +76,8 @@ impl KeygenParticipant {
 
     /// Processes the incoming message given the storage from the protocol
     /// participant (containing auxinfo and keygen artifacts). Optionally
-    /// produces a [KeysharePrivate] and [KeysharePublic] once keygen is
+    /// produces a [`KeysharePrivate`](super::keyshare::KeySharePrivate) and
+    /// [`KeySharePublic`](super::keyshare::KeySharePublic) once keygen is
```

Comment:
note: I build the doc locally with `--document-private-items` and had to fix this to make it work. Should we change the CI to build private docs by default? Most of the crate is private but I think it's good practice in a cryptography library to tend toward good, voluminous documentation.

Reply:
That seems like a reasonable approach to me.

Reply:
Added #79